plugins/hls-haddock-comments-plugin/src/Ide/Plugin/HaddockComments.hs

```diff
@@ -7,7 +7,7 @@
 {-# LANGUAGE RecordWildCards           #-}
 {-# LANGUAGE ViewPatterns              #-}
 
-module Ide.Plugin.HaddockComments (descriptor) where
+module Ide.Plugin.HaddockComments (descriptor, E.Log) where
```

Comment:
Is this new import item used somewhere?

Reply:
Yes, here: https://github.com/haskell/haskell-language-server/pull/3347/files#diff-52fd7f78c5c4a67b8059732e6f4d8216411b28c7d890cfccc01180c1332740a2R21